zilencer/management/commands/render_messages.py

```diff
@@ -47,9 +47,14 @@ def handle(self, *args, **options):
             result.write('[')
             messages = Message.objects.filter(id__gt=latest - amount, id__lte=latest).order_by('id')
             for message in queryset_iterator(messages):
+                content = message.content
+                if message.edit_history:
+                    history = ujson.loads(message.edit_history)
+                    history = sorted(history, key=lambda i: i['timestamp'], reverse=True)
+                    content = history[-1]['prev_content']
```

Comment:
Also, I don't totally understand this logic -- you sorted the messages, reserve them, and then take the last one?  Why not just take the first entry after sorting without the reverse?